src/Core/Prefab/Planar/PlanarLayer.js

```diff
@@ -11,6 +11,7 @@ import PlanarTileBuilder from './PlanarTileBuilder';
  * internally for optimisation.
  */
 class PlanarLayer extends TiledGeometryLayer {
+    #isNewExtent;
```

Comment:
if we decide to merge it without modifying `globalExtentTMS` usage (see my other comment), can you rename to `hasNewExtent` instead please?

Reply:
Ok done !